Makefile

```diff
@@ -70,5 +70,7 @@ promu:
 	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
 	$(GO) get -u github.com/prometheus/promu
 
+frank:  $(shell find frankenstein -name "*.go")
+	go build -o $@ ./frankenstein/cmd/frankenstein
```

Comment:
Not the Prometheus way (that would be adding it to https://github.com/prometheus/prometheus/blob/master/.promu.yml#L6-L9), but works for now.